utils/kata-manager.sh

```diff
@@ -782,6 +821,25 @@ handle_docker()
 	docker --version
 }
 
+handle_nerdctl()
+{
+	{ containerd_installed; ret=$?; } || true
+	if [ "$ret" -eq 0 ]
+	then
+		info "Backing up previous $containerd_project configuration"
+		[ -e "$containerd_config" ] && sudo mv $containerd_config $containerd_config.system-$(date -Iseconds)
+	fi
+
+	install_nerdctl
+
+	configure_containerd "$enable_debug"
```

Comment:
The full nerdctl release contains a bundled containerd, which also includes a `lib/systemd/system/containerd.service`, so this _might_ not work fwics because `configure_containerd()` installs a pristine containerd service, which might differ. A quick check shows that they do differ, but not by much:

```diff
$ diff -u containerd-pristine.service lib/systemd/system/containerd.service                                                                                   work [0]
--- containerd-pristine.service 2024-01-31 09:30:34.343364923 +0000
+++ lib/systemd/system/containerd.service       2023-12-12 11:04:18.000000000 +0000
@@ -18,6 +18,8 @@
 After=network.target local-fs.target

 [Service]
+#uncomment to enable the experimental sbservice (sandboxed) version of containerd/cri integration
+#Environment="ENABLE_CRI_SANDBOXES=sandboxed"
 ExecStartPre=-/sbin/modprobe overlay
 ExecStart=/usr/local/bin/containerd

@@ -26,12 +28,11 @@
 KillMode=process
 Restart=always
 RestartSec=5
-
 # Having non-zero Limit*s causes performance problems due to accounting overhead
 # in the kernel. We recommend using cgroups to do container-local accounting.
 LimitNPROC=infinity
 LimitCORE=infinity
-
+LimitNOFILE=infinity
 # Comment TasksMax if your systemd version does not supports it.
 # Only systemd 226 and above support this version.
 TasksMax=infinity
```

So, we could either leave the logic as-is, or copy / link `/usr/local/lib/systemd/system/containerd.service` to `/etc/systemd/system/containerd.service`, to ensure the nerdctl version of the service file is used? I guess it shouldn't make a difference, but might be worth adding a comment explaining some of these details.